pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -160,3 +175,160 @@ def step(self, t, dt, dtau):
 
         # Return the index of the bank containing u(n+1,m+1)
         return r1, r0
+
+
+class DualRKVdH2RStepper(BaseDualPseudoStepper):
+    # Coefficients
+    a = []
+    b = []
+    bhat = []
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Compute the c and error coeffs
+        self.c = [0.0] + [sum(self.b[:i]) + ai for i, ai in enumerate(self.a)]
```

Comment:
Some of this code is generic to any dual stepper with an error estimator.  Factor accordingly.

Reply:
This coefficient work is specific to VdH type schemes.

Reply:
Missed this comment. Fixing it.